pkg/app/master/commands/cliprompt.go

```diff
@@ -79,7 +79,7 @@ func (ia *InteractiveApp) execute(command string) {
 	}
 
 	if parts[0] == "exit" {
-		app.ShowCommunityInfo()
+		app.ShowCommunityInfo(FlagConsoleOutput)
```

Comment:
is it the flag name passed here or the actual flag value?

Reply:
actual flag value